HW-2 Kafka

noslav7 opened the pull request.

noslav7 committed:
HW-2 Kafka

Alexandr056 commented on src/main/java/com/example/aspect_oriented_programming/kafka/KafkaConsumerService.java:
```diff
@@ -0,0 +1,23 @@
+    }
+
+    @KafkaListener(topics = "test-topic", groupId = "task-service")
+    private void listen(String message) {
```
строка? стоит делать DTO 
читать не по 1 сообщению, это оптимальнее
ack проигнорировали, а все же он достоит внимания ручками отправлять acknowledge

Alexandr056 reviewed (approved):
вариант рабочий но "на скорую руку", нет логов, нет обработки возможных ошибок + пара комментов

noslav7 committed:
HW-2 Kafka fix
- modified:
application.properties
KafkaConsumerService
KafkaProducerService
TaskService
- created:
KafkaConfig
KafkaMessageDTO

noslav7 committed:
HW-2 Kafka fix
- modified:
application.properties
KafkaConsumerService
KafkaProducerService
TaskService
- created:
KafkaConfig
KafkaMessageDTO

noslav7 committed:
Merge remote-tracking branch 'origin/feature/kafka' into feature/kafka

# Conflicts:
#	src/main/resources/application.properties

noslav7 committed:
HW-2 Kafka fix
- modified:
application.properties
KafkaConsumerService
KafkaProducerService
TaskService
- created:
KafkaConfig
KafkaMessageDTO
